packages/core/useTextDirection/index.test.ts

```diff
@@ -0,0 +1,74 @@
+import { afterEach, describe, expect, it } from 'vitest'
+import { useTextDirection } from '.'
+
+describe('useTextDirection', () => {
+  const htmlEl = document.documentElement
+  const bodyEl = document.body
+
+  afterEach(() => {
+    htmlEl.removeAttribute('dir')
+    document.body.removeAttribute('dir')
+  })
+
+  it('should be defined', () => {
+    expect(useTextDirection).toBeDefined()
+  })
+
+  it('should have default value of ltr', () => {
+    const dir = useTextDirection()
+    expect(dir.value).toBe('ltr')
+    expect(htmlEl.attributes).toMatchInlineSnapshot(`NamedNodeMap {}`)
+  })
+
+  it('should take initialValue as default if passed', () => {
+    const dir = useTextDirection({ initialValue: 'rtl' })
+    expect(dir.value).toBe('rtl')
+    expect(htmlEl.attributes).toMatchInlineSnapshot(`NamedNodeMap {}`)
```

Comment:
not sure if this is intended.

Reply:
Yeah I think it is. By default it's a reactive "read" and does not write. `initialValue` is more like a fallback value on reading.